src/__tests__/helpers/test-utils.tsx

```
@@ -0,0 +1,164 @@
+import React from 'react'
+import { render, RenderOptions } from '@testing-library/react'
```

ESLint

[error] `@testing-library/react` import should occur before import of `react`

(import/order)